<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite guice-stage="DEVELOPMENT" verbose="0" name="Failed suite [Suite]">
  <test name="Test">
    <classes>
      <class name="com.letgo.tests.basetests.TestngTest"/>
      <class name="com.letgo.tests.testcases.TestCase1"/>
      <class name="com.letgo.tests.testcases.TestCase2"/>
    </classes>
  </test> <!-- Test -->
  <test name="Test(failed)">
    <classes>
      <class name="com.letgo.tests.basetests.TestngTest">
        <methods>
          <include name="nullPointerExseptionShouldPass"/>
          <include name="shouldAlwaysPass"/>
        </methods>
      </class> <!-- com.letgo.tests.basetests.TestngTest -->
      <class name="com.letgo.tests.testcases.TestCase2">
        <methods>
          <include name="tearDown"/>
          <include name="searchForItemInWrongGroup"/>
          <include name="setUp"/>
        </methods>
      </class> <!-- com.letgo.tests.testcases.TestCase2 -->
      <class name="com.letgo.tests.testcases.TestCase1">
        <methods>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="signUp"/>
        </methods>
      </class> <!-- com.letgo.tests.testcases.TestCase1 -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
